### PR TITLE
Update the game over message.

### PR DESCRIPTION
**Pull request by DavidNiz-A**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by DavidNiz-A — COMMENTED** [review]
Que se yo. No sé que decirte...

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (DavidNizDev) [inline index.html:50]: Sugiriendo cambios ndeah.

File: .github/steps/-step.txt [hunk 2]
```diff
@@ -1 +1 @@
-0
+4
```
Comment (DavidNizDev) [inline .github/steps/-step.txt]:
```suggestion
5
```

File: index.html (hunk 1 above)
Comment (DavidNiz-A) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again! 🧑‍💻 🤖!</h2>
```

**Pull request merged**